index.html

```diff
@@ -8,26 +8,35 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <nav>
-      <ul>
-        <li><a href="#" class="welcome">Welcome</a></li>
-        <li><img src="img/Union.png" alt="menu bar" /></li>
-      </ul>       
-    </nav>
+        <header>
+                <div><a href="#" class="welcome">Welcome</a></div>
+               <nav>
+                 <ul>
+                   <li><a class="menu-bar" href="#">Portfolio</a></li>
+                   <li><a class="menu-bar" href="#">About</a></li>
+                   <li><a class="menu-bar" href="#">Contact</a></li>   
+                 </ul>       
+               </nav>
+               <div class="menu-image">
+                        <img class="menu-icon"src="img/Union.png" alt="menu bar">
+                </div>
+                <div> <img class="af image-a"src="img/af.png" alt="home image">
+                </div>
 
-    <section class="myname">
+        </header>
+
+    <section class="back-pic">
+      <section class="myname">
       <h1 class="introduction">
-        Hey there. I’m Chukwuemeka <br />
+        Hey there. I’m Chukwuemeka 
         <span> I’m a software developer</span>
       </h1>
       <p class="description">
         I can help you build a product , feature or website Look through some of
-        my work and experience! <br />If you like what you see and have <br />a
-        project you need coded, <br />
-        don’t hestiate to contact me.
+        my work and experience! If you like what you see and have a
+        project you need coded, don’t hesitate to contact me.
       </p>
```

Comment:
- [ ] Good job so far 👏 I think it would be better if the headline had the `background-image` similar to the one in Figma, in Figma the background looks bigger and it has lines going to the center, kindly try to add it. Also, please try to put the parts of the header that had different colors in separate lines (blue text on top and green text on the bottom) and kindly increase the `font-size` of this header to match the design 💯 Lastly, please center the button below the social media links 👍  

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206474007-9cab1bbd-76e3-453b-b85a-14383cb061a5.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206474078-8595d290-c0ce-4702-9efc-3dd8bc3e0ca4.png)

Reply:
If I use pixel sizes , it will scatter in my mobile view. I feel that using % is better